modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -182,23 +154,22 @@ describe('test tss helper functions', function () {
     });
 
     it('should fail if common keychains do not match', async function () {
-      const bitgoToUserShare = await encryptNShare({
-        keyShare: bitgoKeyShare,
-        recipientIndex: 1,
-        recipientGpgPublicArmor: userGpgKeypair.publicKey,
-        senderGpgPrivateArmor: bitgoGpgKeypair.privateKey,
-      });
-      const backupToUserShare = await encryptNShare({
-        keyShare: backupKeyShare,
-        recipientIndex: 1,
-        recipientGpgPublicArmor: userGpgKeypair.publicKey,
-        senderGpgPrivateArmor: backupGpgKeypair.privateKey,
-      });
-
-      await createCombinedKey({
-        keyShare: userKeyShare,
-        commonKeychain: 'nottherightkeychain',
-        encryptedNShares: [{
+      const bitgoToUserShare = await encryptNShare(
+        bitgoKeyShare,
+        1,
+        userGpgKeypair.publicKey,
+        bitgoGpgKeypair.privateKey,
+      );
+      const backupToUserShare = await encryptNShare(
+        backupKeyShare,
+        1,
+        userGpgKeypair.publicKey,
+        backupGpgKeypair.privateKey,
+      );
+
+      await createCombinedKey(
+        userKeyShare,
```

Comment:
How did userKeyShare didn't match?, did we mutate this in a previous testcase?, probably lets all the mutation within the testcase itself to have give it more clarity

Reply:
this should fail to combine the keys because we pass in invalid common key chain